test/com/amazon/ion/impl/bin/IonEncoder_1_1Test.java

```diff
@@ -72,7 +86,7 @@ public void testWriteNullValueForDatagram() {
             "true, 5E",
             "false, 5F",
     })
-    public void testWriteBooleanValue(Boolean value, String expectedBytes) {
+    public void testWriteBooleanValue(boolean value, String expectedBytes) {
```

Comment:
This is an incidental change. Then `writeBooleanValue` method accepts `boolean`, so I made the test do the same.